api_tests/dry/rfc003/test.js

```diff
@@ -147,29 +147,29 @@ describe("RFC003 HTTP API", () => {
                 },
                 alpha_asset: {
                     name: alpha_asset_name,
-                    quantity: alpha_asset_quantity,
+                    quantity: "100",
```

Comment:
🔧What do you think of storing this in a variable so that you can reference it lateron instead of needing to hardcode `100` ?

Reply:
Yes, good idea :wink: